src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
-body {
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    margin-top: 33px;
+
+    &-photo {
+      max-width: 160px;
+      max-height: 134px;
+      object-fit: contain;
+    }
```

Comment:
According to the BEM methodology, an element should not be nested inside another element. Here, `&__image-photo` is an element nested inside `&__image`, which is also an element. Consider refactoring to follow BEM conventions, perhaps by using a modifier or a block instead.